var/spack/repos/builtin/packages/mdtest/package.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Mdtest(Package):
+    """mdtest is an MPI-coordinated metadata benchmark test 
+       that performs open/stat/close operations on files 
+       and directories and then reports the performance."""
+
+    homepage = "https://github.com/LLNL/mdtest"
+
+    version('1.9.3', git='https://github.com/LLNL/mdtest.git', commit='49f3f0')
+
+    depends_on('mpi')
+
+    def install(self, spec, prefix):
+        filter_file(r'\$\(CC\.\$\(OS\)\)',  spec['mpi'].mpicc,  'Makefile')
```

Comment:
If you use the kwarg `string=True`, it will prevent Spack from interpreting your search string as a regular expression. Then you won't need all of the backslashes.